### PR TITLE
Narrative and credits scene

### PR DESCRIPTION
**Pull request by AlejandroPla**

### CODE REVIEW

File: dialog/dialog2.txt [hunk 1]
```diff
@@ -1,2 +1,16 @@
...
-Narrator Portraits-Narrator Continue .
+15
+Narrator Portraits-Narrator Thassa's men inspected the castle. Finally, they stood up in front of the main hall door. .
+Valar Portraits-Valar Your majesty, we found where they are! .
+Narrator Portraits-Narrator Thassa ran through the passages guided by Valar. Behind her, a small group of her forces were following her. .
+Narrator Portraits-Narrator The first glance she had was petrifying. Behind the enemy line, a sorceress could be seen with a sharp dagger in the king's throat. .
+Thassa Portraits-Thassa FATHER! NOO! .
+Narrator Portraits-Narrator One clean move was enough to sink Thassa into despair. Blood was flowing through the killer hands. Her father, the King, was dead. .
+Narrator Portraits-Narrator Thassa, full of anger, went on her knees while her troops went on the attack to protect their dead king. .
+Thassa Portraits-Thassa ... No.. My father is... argh... no... .
```
Comment (kyranet) [inline dialog/dialog2.txt]:
```suggestion
Thassa Portraits-Thassa ... No... My father is... argh... no... .
```

File: dialog/dialog2.txt [hunk 2]
```diff
@@ -1,2 +1,16 @@
...
+Narrator Portraits-Narrator She  mumbled a few curse words, and stood up again, although there were tears on her face, the looked fierce and ready to kill. .
```
Comment (kyranet) [inline dialog/dialog2.txt]:
```suggestion
Narrator Portraits-Narrator She mumbled a few curse words, and stood up again, although there were tears on her face, the looked fierce and ready to kill. .
```

File: dialog/dialog2.txt [hunk 3]
```diff
@@ -1,2 +1,16 @@
...
+Thassa Portraits-Thassa If this is you want, then war is you shall have! .
```
Comment (kyranet) [inline dialog/dialog2.txt]:
```suggestion
Thassa Portraits-Thassa If this is what you want, then war is what you shall have! .
```

File: dialog/dialog3transition.txt [hunk 4]
```diff
@@ -1,3 +1,31 @@
...
\ No newline at end of file
+29
+Narrator Portraits-Narrator Thassa's forces fought fiercely and won the battle. The mortally wounded sorceress was interrogated by Thassa. Thassa landed multiple blows on Sheissah's body, but she did not loosen her mouth even one bit. .
+Thassa Portraits-Thassa Gods!... Who is commanding you? Why are your forces attacking the realm?! .
+Sorcerer Portraits-MageEnemy Your grace... I am known throughout this land as Sheissah, the Bane of the Living. .
+Narrator Portraits-Narrator A malicious smile was hiding in Sheissah's visage. .
+Thassa Portraits-Thassa You did not answer what I asked. .
+Narrator Portraits-Narrator Another stroke, this time on her face. .
+Thassa Portraits-Thassa Who and where is your leader? .
+Narrator Portraits-Narrator No answer.Thassa, consumed by her anger, continued landing strokes all over Sheissah's half-dead body. She made sure Sheissah could not die by telling her soldiers to immobilize her and treating her bleeding wounds.
```
Comment (kyranet) [inline dialog/dialog3transition.txt]:
```suggestion
Narrator Portraits-Narrator No answer. Thassa, consumed by her anger, continued landing strokes all over Sheissah's half-dead body. She made sure Sheissah could not die by telling her soldiers to immobilize her and treating her bleeding wounds. .
```

File: dialog/dialog3transition.txt [hunk 5]
```diff
@@ -1,3 +1,31 @@
-2
-Thassa Portraits-Thassa Thanks for playing this demo! .
-Narrator Portraits-Narrator To be continued... .
\ No newline at end of file
+29
```
Comment (kyranet) [inline dialog/dialog3transition.txt]:
```suggestion
30
```

File: dialog/dialog3transition.txt [hunk 6]
```diff
@@ -1,3 +1,31 @@
...
+Thassa Portraits-Thassa Gods!... Who is commanding you? Why are your forces attacking the realm?! .
+Sorcerer Portraits-MageEnemy Your grace... I am known throughout this land as Sheissah, the Bane of the Living. .
+Narrator Portraits-Narrator A malicious smile was hiding in Sheissah's visage. .
+Thassa Portraits-Thassa You did not answer what I asked. .
+Narrator Portraits-Narrator Another stroke, this time on her face. .
+Thassa Portraits-Thassa Who and where is your leader? .
+Narrator Portraits-Narrator No answer.Thassa, consumed by her anger, continued landing strokes all over Sheissah's half-dead body. She made sure Sheissah could not die by telling her soldiers to immobilize her and treating her bleeding wounds.
+Narrator Portraits-Narrator Sheissah, seeing she could not escape nor die, gave up. .
+Sheissah Portraits-MageEnemy Stop... I cannot take this any longer... Please... Let me die in peace... .
+Sheissah Portraits-MageEnemy ...I...I was requested to take Deiro's Gem for my lord... .
```
Comment (kyranet) [inline dialog/dialog3transition.txt]:
```suggestion
Sheissah Portraits-MageEnemy I... I... was requested to take Deiro's Gem for my lord... .
```

File: dialog/dialog3transition.txt [hunk 7]
```diff
@@ -1,3 +1,31 @@
...
+Thassa Portraits-Thassa Who is your lord?! .
+Narrator Portraits-Narrator She was just about to speak, when Valar walked ahead of Thassa, drew his sword and sliced Sheissah's throat. Sheissah, agonizing, tried to murmure some words out but died before she could. .
+Thassa Portraits-Thassa Valar?! Why?! .
+Valar Portraits-Valar I am sorry, your Grace, I-I couldn't hold myself any longer. Just this pain, I can't, I am sorry. .
+Narrator Portraits-Narrator Thassa, astonished, verged the king's corpse wrapping it with her cap. .
+Thassa Portraits-Thassa Did you understand what she tried to say? .
+Valar Portraits-Valar I have no idea your majesty. .
+Narrator Portraits-Narrator She muttered some intelligible words and faced the other way .
```
Comment (kyranet) [inline dialog/dialog3transition.txt]:
```suggestion
Narrator Portraits-Narrator She muttered some intelligible words and faced the other way. .
```

File: dialog/dialog3transition.txt [hunk 8]
```diff
@@ -1,3 +1,31 @@
...
+Narrator Portraits-Narrator Hopelessly, Thassa fell on her knees and collapsed in tears, facing the situation that concerned everyone. Valar approached and placed his hand soaked with Sheissah's blood on Thassa's shoulder. .
+Valar Portraits-Valar Your grace, we no longer have time for this. I have been concerned about the odd mist that is surrounding Crihendrath. As far as I am concerned, it may be caused by the dead horde. .
+Valar Portraits-Valar I would advise sending some of our scouts to fully investigate what is happening in our land. We cannot let the enemy have the tactical advantage any longer. .
+Narrator Portraits-Narrator Thassa agreed and ordered to fully secure the city and afterwards she would personally direct the retinue. .
+Narrator Portraits-Narrator The remnants of the battle were annihilated and the city began an exhaustive reparation while Thassa retinue prepared to depart in search of valuable information that may be useful against the enemies who attacked the reign�s peace and prosperity. .
```
Comment (kyranet) [inline dialog/dialog3transition.txt]:
```suggestion
Narrator Portraits-Narrator The remnants of the battle were annihilated and the city began an exhaustive reparation while Thassa retinue prepared to depart in search of valuable information that may be useful against the enemies who attacked the reigns of peace and prosperity. .
```

File: dialog/dialog4transition.txt [hunk 9]
```diff
@@ -1,2 +1,6 @@
-1
-Thassa Portraits-Thassa Placeholder .
\ No newline at end of file
+5
+Narrator Portraits-Narrator With Drakhtar's army retreating, Thassa realized something: The fog only affected her soldiers and not the undead. .
+Thassa Portraits-Thassa Shite... We can't stay in this mist any longer! We have to leave as soon as possible! Fall back! .
```
Comment (kyranet) [inline dialog/dialog4transition.txt]:
```suggestion
Thassa Portraits-Thassa Oh no... We can't stay in this mist any longer! We have to leave as soon as possible! Fall back! .
```

File: dialog/dialog5.txt [hunk 10]
```diff
@@ -1,2 +1,6 @@
-1
-Thassa Portraits-Thassa Placeholder .
\ No newline at end of file
+5
+Abeizhul Portraits-SoldierEnemy You will regret setting a foot here! Fiends of hell of darkness, come to me! Tonight we are having a feast of blood and souls! .
```
Comment (kyranet) [inline dialog/dialog5.txt]:
```suggestion
Abeizhul Portraits-SoldierEnemy You will regret setting a foot here! Friends of hell and darkness, come to me! Tonight we are having a feast of blood and souls! .
```

File: dialog/dialog5transition.txt [hunk 11]
```diff
@@ -1,2 +1,16 @@
...
+Abeizhul Portraits-SoldierEnemy I am Abeizhul, War, Famine, Death. I come to claim this as my kingdom. As for you... .
+Abeizhul Portraits-SoldierEnemy We will meet again soon, princess of Drakhtar. For now, I take my leave. .
+Narrator Portraits-Narrator The demon opened a sinister portal with his black magic, and in a flash of red and black, he was gone in an instant. Thassa chased after him, just for the portal to disappear right before her eyes. .
+Narrator Portraits-Narrator Thassa thought of what to do next. When she was a kid, she read in books about the circle of Aryassa, a group of magicians, masters in pretty much every type of magic. .
+Narrator Portraits-Narrator She was sure they had to know how to remove or avoid the strange fog that surrounded Crihendrath. They were her only hope. .
+Narrator Portraits-Narrator Thassa invited the circle of magicians of Aryassa to her court and asked for help to pass the mist of death. After a long negotiation, they accepted and agreed to follow her. .
+Narrator Portraits-Narrator Korvuk, the circle's leader, summoned strange looking golems that joined Thassa�s ranks and opened a portal that led next to the enemy fortress. .
```
Comment (kyranet) [inline dialog/dialog5transition.txt]:
```suggestion
Narrator Portraits-Narrator Korvuk, the circle's leader, summoned strange looking golems that joined Thassa's ranks and opened a portal that led next to the enemy fortress. .
```

**Comment by Dankgard** [comment]
> 
> 
> And also, the credits show the names too far apart between each other.

**Pull request merged**